runPerftool.psm1

```diff
@@ -209,14 +218,19 @@ Function ProcessCommands{
     [Parameter(Mandatory=$True)]  [string]$DestIp,
     [Parameter(Mandatory=$True)] [string]$SrcIp,
     [Parameter(Mandatory=$True)]  [string]$CommandsDir,
+    [Parameter(ParameterSetName='PassAuth', Mandatory=$False)]  [bool]$PassAuth = $False,
     [Parameter(Mandatory=$True, Position=0, HelpMessage="Dest Machine Username?")]
     [string] $DestIpUserName,
     [Parameter(Mandatory=$True, Position=0, HelpMessage="Dest Machine Password?")]
     [SecureString]$DestIpPassword,
+    [Parameter(Mandatory=$False, Position=0, HelpMessage="Dest Machine Key File?")]
+    [String]$DestIpKeyFile = "",
     [Parameter(Mandatory=$True, Position=0, HelpMessage="Src Machine Username?")]
     [string] $SrcIpUserName,
     [Parameter(Mandatory=$True, Position=0, HelpMessage="Src Machine Password?")]
     [SecureString]$SrcIpPassword,
+    [Parameter(Mandatory=$False, Position=0, HelpMessage="Src Machine Key File?")]
+    [String]$SrcIpKeyFile = "",
     [Parameter(Mandatory=$True, Position=0, HelpMessage="Test Machine Username?")]
     [string] $TestUserName,
```

Comment:
Is this only used for building the path?

Reply:
Yes, for the path to the private key that will give access to the remote VMs.